stackslib/src/chainstate/stacks/boot/mod.rs

```diff
@@ -624,7 +647,7 @@ impl StacksChainState {
 
         let mut signer_set = BTreeMap::new();
         for entry in entries.iter() {
-            let signing_key = if let Some(PrincipalData::Standard(s)) = entry.signing_key.clone() {
+            let signing_key = if let Some(PrincipalData::Standard(s)) = entry.signer.clone() {
```

Comment:
This wouldn't be a `signing_key` anymore ;)  Maybe a better variable name like `signing_addr`?